utils/graphql.ts

```diff
@@ -2,14 +2,14 @@ import { GraphQLClient } from "graphql-request";
 
 export let gqlClient: GraphQLClient
 
-export const initializeGqlClient = () => {
+export const initializeGqlClient = (useLocalApi: string | undefined) => {
     if (gqlClient) {
         return gqlClient
     }
 
     let apiURL
 
-    if (process.env.NODE_ENV == 'development') {
+    if (useLocalApi === "development") {
```

Comment:
this is ....confusing... let's set useLocalApi to true/false instead of strings.